src/summariser.py

```diff
@@ -2,17 +2,48 @@
 from cohere.types.non_streamed_chat_response import NonStreamedChatResponse
 from src.utils import get_cohere_api_key
 import logging
+from langchain_community.document_loaders import WebBaseLoader
+import re
+from langchain_core.documents.base import Document
 
 logger = logging.getLogger(__name__)
 
 
+def extract_urls(text):
+    url_pattern = re.compile(r"(https?://\S+|www\.\S+)")
+    urls = url_pattern.findall(text)
+    return urls
+
+
 def read_prompt_header() -> str:
     # Read the prompt from the file
     with open("src/prompt.txt", "r") as file:
         prompt_header = file.read()
     return prompt_header
 
 
+def langchain_webcontent(message: str) -> str:
+    """Load the text from the url stored inside message
+
+    Args:
+        message (str): _description_
```

Comment:
Need to fill in the description! Seems like you actually already added the extension I recommended above (congrats!) but it needs a bit more :)